doc/hcr.rst

```diff
@@ -26,59 +26,61 @@ To install SDL2 you can use ``nimble install sdl2``.
 .. code-block:: nim
 
   # logic.nim
-  import sdl2/sdl
-
+  import sdl2
+  
   #*** import the hotcodereloading stdlib module ***
   import hotcodereloading
-
+  
   var runGame*: bool = true
-  var window: Window
-  var renderer: Renderer
-
+  var window: WindowPtr
+  var renderer: RendererPtr
+  var evt = sdl2.defaultEvent
+  
   proc init*() =
-    discard sdl.init(INIT_EVERYTHING)
-    window = createWindow("testing", WINDOWPOS_UNDEFINED.cint, WINDOWPOS_UNDEFINED.cint, 640, 480, 0'u32)
-    assert(window != nil, $sdl.getError())
+    discard sdl2.init(INIT_EVERYTHING)
+    window = createWindow("testing", SDL_WINDOWPOS_UNDEFINED.cint, SDL_WINDOWPOS_UNDEFINED.cint, 640, 480, 0'u32)
+    assert(window != nil, $sdl2.getError())
     renderer = createRenderer(window, -1, RENDERER_SOFTWARE)
-    assert(renderer != nil, $sdl.getError())
-
+    assert(renderer != nil, $sdl2.getError())
+  
   proc destroy*() =
     destroyRenderer(renderer)
     destroyWindow(window)
-
-  var posX = 1
-  var posY = 0
-  var dX = 1
-  var dY = 1
-
+  
+  var posX : cint = 1
+  var posY : cint = 0
+  var dX : cint = 1
+  var dY : cint = 1
```

Comment:
```suggestion
  var posX: cint = 1
  var posY: cint = 0
  var dX: cint = 1
  var dY: cint = 1
```

Some minor cosmetic changes.

Reply:
commit `daccd8464b4186013ef5b59d13b3939512e2fdc1` resolved this